app/pages/adduser/controller.js

```diff
@@ -1,33 +1,35 @@
-// import axios from 'axios';
+// import axios from 'axios'
+import { getData } from '../../apiProvider';
 import { getContext } from './contextCreator';
-import { identityServer } from '../../config';
+import { organisationApiUrl } from '../../config';
 import { logger } from '../../logger';
 
-export const getAddUserContext = (orgId) => {
-  const endpoint = `${identityServer}/api/v1/Organisations/${orgId}`;
-  logger.info(`api called: [GET] ${endpoint}`);
-  logger.info(`Organisation ${orgId} returned`);
-  return getContext({ data: {} });
+export const getAddUserContext = async ({ organisationId, accessToken }) => {
+  const options = { organisationId };
+  // Need to call this to get org name
```

Comment:
can we remove this comment also?

Reply:
No because it took me a few minutes to work out why we were calling this so I put it in to save everyone that job ie what comments should be used for 😝

Reply:
👍